pkg/render/logstorage.go

```diff
@@ -1479,13 +1462,13 @@ func (es elasticsearchComponent) oidcUserRole() []client.Object {
 					APIGroups:     []string{""},
 					Resources:     []string{"configmaps"},
 					ResourceNames: []string{OIDCUsersConfigMapName},
-					Verbs:         []string{"get", "list", "watch"},
+					Verbs:         []string{"*"},
```

Comment:
We should specify the Verbs explicitly instead of using the wildcard. We haven't been very consistent about this so far but that is what we should be striving for.